doc/cascadia/profiles.schema.json

```diff
@@ -509,6 +509,10 @@
           ],
           "type": "string"
         },
+        "experimental.retroTerminalEffect": {
+          "description": "When set to true, enable retro terminal effects.",
```

Comment:
"This is an experimental feature, and its continued existence is not guaranteed."